src/olympia/templates/base.html

```diff
@@ -56,6 +56,17 @@
         {% block bodyattrs %}{% endblock %}>
 
     <div id="main-wrapper">
+      {% if user.is_authenticated and user.is_survey_eligible(survey_id=amo.DEV_EXP_SURVEY_ALCHEMER_ID) and switch_is_active('dev_exp_survey') %}
```

Comment:
We don't want to add that extra database query to every reviewer page. Can you move that block to `devhub/base.html` ? You can re-use the `{% block site_notice %}` to insert the survey right where you had it, and add a `{{ block.super }}` to keep the site notice logic still.